source/code/plugins/VMInsightsDataCollector.rb

```diff
@@ -354,8 +346,12 @@ def get_up_net_devices
 
         class Fs
             def initialize(device_name, mount_point, size_in_bytes, free_space_in_bytes)
-                raise ArgumentError, device_name unless device_name.start_with?("/dev/")
                 raise ArgumentError, mount_point unless mount_point.start_with? "/"
+                ind = device_name.index('/', 1)
```

Comment:
I don't see a test case where `ind` will be nil. Is that because the value of `device_name` must always begin with "/dev/"? If so, simplify the code.